src/middleware/secure-file-storage-api/src/main/resources/application.properties

```diff
@@ -0,0 +1,4 @@
+aws.access.key.id=AKIA2DFZL3BE2THTJHID
```

Comment:
change to environment variables and we can share the creds outside of github

Reply:
Done!